svgdigitizer/__main__.py

```diff
@@ -128,8 +128,13 @@ def plot(svg, sampling_interval):
     default=None,
     help="write output files to this directory",
 )
+@click.option('--algorithm',
+              type=click.Choice(['axis-aligned', 'mark-aligned'], case_sensitive=False),
+              default='axis-aligned',
+              help="For orthogonal plot axes use 'axis-aligned' (default). For skewed axis use 'mark-aligned'."
+)
```

Comment:
```suggestion
@click.option('--skewed',
              is_flag=True
              help="If the plot axes are skewed use '--skewed-axes' to align the axes along the markers instead of orthogonally (default)."
)
Does this work if we have a point and a scalebar for the axis?
```

Reply:
Is using the skewed flag easier to understand for the user?

Reply:
I really like this idea! I don't think we will add so many algorithms in the near future.

Reply:
```patch
-If the plot axes are skewed use '--skewed' to align the axes along the markers instead of orthogonally (default).
+Detect non-orthogonal skewed axes going through the markers instead of assuming that axes are perfectly horizontal and vertical.
```

Sounds good to me. Maybe we should not repeat the keyword in its own help string?

Reply:
> 

The program does not crash with a scalebar, however, the output plot is not correct. I'll add it to the list of issues.